gateway/src/apicast/configuration.lua

```diff
@@ -163,9 +164,15 @@ function _M.filter_services(services, subset)
     if service:match_host(service_regexp_filter) or subset[service.id] then
       insert(selected_services, service)
     else
-      ngx.log(ngx.WARN, 'filtering out service ', service.id)
+      table.insert(filtered_services, service.id)
```

Comment:
We cache the table.insert at the top of the file so it won't need to be load again for each call. Hence, we should use `insert(filtered_services, service.id)` here instead.

Reply:
Thanks!